velox/exec/HashBuild.h

```diff
@@ -311,6 +312,39 @@ inline std::ostream& operator<<(std::ostream& os, HashBuild::State state) {
   os << HashBuild::stateName(state);
   return os;
 }
+
+class HashBuildSpiller : public SpillerBase {
+ public:
+  static constexpr std::string_view kType = "HashBuildSpiller";
+
+  HashBuildSpiller(
+      core::JoinType joinType,
+      RowContainer* container,
+      RowTypePtr rowType,
+      HashBitRange bits,
+      const common::SpillConfig* spillConfig,
+      folly::Synchronized<common::SpillStats>* spillStats);
+
+  /// Invoked to spill the current hash table in container.
+  void spill();
+
+  /// Invoked to spill input vector in 'spillVector'.
```

Comment:
This could be a base method and build and probe have different implementations? thanks!

```
/// Invoked to spill input vector from a given partition.
```